Gemfile.jruby-2.3.lock.release

```diff
@@ -529,21 +528,21 @@ GEM
       rack
     rack-test (0.7.0)
       rack (>= 1.0, < 3)
-    rake (12.1.0)
+    rake (12.2.1)
```

Comment:
This one makes me nervous. https://github.com/elastic/logstash/issues/8220 and https://github.com/elastic/logstash/issues/8214 are tangentially related to the rake version (i.e. smoke - no fire). In the spirit of pinning well known working dependency graph, I would suggest to keep this pinned 12.1.0 until 6.1.

Reply:
I'm +1 on this. In fact, I'm thinking that I'm actually +1 on *not* bumping any gems between RC2 and 6.0.0. This would be a change to our release process, but I think it makes sense. WDYT?

Reply:
My thoughts...
I see RC->Release == patch release, at least the same rules should apply.

IMO patch release rules:

Plugins:  New plugin patch version's and (case-by-case) minor plugin updates are good. They can always downgrade those if there is the bug. The exception here is if a patch/minor upgrade pinned a new transitive version. 

For non-plugins gems, I would think to _not_ bump them unless they are intentionally getting upgraded. For example in this change set there the Jackson version bump is intentional and should be in 6.0.0, as it fixes a regression https://github.com/elastic/logstash/issues/8439

Reply:
I think the tricky thing is there is no `bundle update --only-logstash-plugins-and-other-things-that-matter`.

Thinking about it, I don't think this PR is the right place to have this discussion (even though I started it!), and we should debate this policy going forward, but not right now.

WDYT? @jordansissel can you chime in here?